src/index.d.ts

```diff
@@ -17,7 +17,10 @@ declare namespace uvu {
 }
 
 type Context = Record<string, any>;
-export const test: uvu.Test<Context>;
+
+export type Test<T=Context> = uvu.Test<T>;
```

Comment:
I didn't have this exposed because of the generic naming. 
But now that it is, what do you think about using the `Suite` name instead, since that's what it is?

Reply:
I mean you could argue the same thing with `Callback`. To me the suite, is the collection of test's (at least mentally), and a Test is the individual units. Seeing as `suite()` returns a uvu.Test it's only fitting to also export that type.

Its like you don't share suites, you share tests. I know its the same thing—but i still believe the naming her is right.